src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -0,0 +1,590 @@
+        except RuntimeError:
+            logging.warn("Error while killing jobs on the schedd: WMAgent_RequestName=%s", workflow)
+
+        return
+
+    def updateJobInformation(self, workflow, task, **kwargs):
+        """
+        _updateJobInformation_
+
+        Update job information for all jobs in the workflow and task,
+        the change will take effect if the job is Idle or becomes idle.
+
+        The currently supported changes are only priority for which both the task (taskPriority)
+        and workflow priority (requestPriority) must be provided.
+        """
+        schedd = htcondor.Schedd()
+
+        if 'taskPriority' in kwargs and 'requestPriority' in kwargs:
+            newPriority = int(kwargs['requestPriority']) + int(kwargs['taskPriority'] * self.maxTaskPriority)
+            try:
+                constraint = "WMAgent_SubTaskName =?= %s" % classad.quote(task)
+                constraint += " && WMAgent_RequestName =?= %s" % classad.quote(workflow)
+                constraint += " && JobPrio =!= %d" % newPriority
+                schedd.edit(constraint, 'JobPrio', classad.Literal(newPriority))
+            except:
+                logging.warn("Failed to update JobPrio for WMAgent_SubTaskName=%s", task)
+
+        return
+
+    def getClusterAd(self):
+        """
+        _initSubmit_
+
+        Return common cluster classad
+
+        scriptFile & Output/Error/Log filenames shortened to
+        avoid condorg submission errors from >256 chars paths
+
+        """
+        ad = classad.ClassAd()
+
+        #ad['universe'] = "vanilla"
+        ad['Requirements'] = classad.ExprTree(self.reqStr)
+        ad['ShouldTransferFiles'] = "YES"
+        ad['WhenToTransferOutput'] = "ON_EXIT"
+        ad['UserLogUseXML'] = True
+        ad['JobNotification'] = 0
+        ad['Core'] = 0
+        ad['Cmd'] = self.scriptFile
+        ad['Out'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".out")')
+        ad['Err'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".err")')
+        ad['UserLog'] = classad.ExprTree('strcat("condor.", ClusterId, ".", ProcId, ".log")')
+
+        ad['WMAgent_AgentName'] = self.agent
+
+        ad['JOBGLIDEIN_CMSSite'] = classad.ExprTree('isUndefined(GLIDEIN_CMSSite) ? Unknown : GLIDEIN_CMSSite')
+
+        ad['JobLeaseDuration'] = classad.ExprTree('isUndefined(MachineAttrMaxHibernateTime0) ? 1200 : MachineAttrMaxHibernateTime0')
+
+        # Required for global pool accounting
+        ad['AcctGroup'] = self.acctGroup
+        ad['AcctGroupUser'] = self.acctGroupUser
+
+        # Customized classAds for this plugin
+        ad['DESIRED_Archs'] = "INTEL,X86_64"
+
+        ad['x509userproxy'] = self.x509userproxy
+        ad['x509userproxysubject'] = self.x509userproxysubject
+
+        ad['Rank'] = 0.0
+        ad['TransferIn'] = False
+
+        params_to_add = htcondor.param['SUBMIT_ATTRS'].split() + htcondor.param['SUBMIT_EXPRS'].split()
```

Comment:
More interesting here is params_to_skip, parameters that the schedd will set for you but it doesn't really need to since we already set them ourselves. Bug, historic leftovers or why are they there ?